src/Components/AddProduct.tsx

```diff
@@ -1,101 +1,274 @@
-
-          </>
-
-
-        </div>
-        <div style={{ display: 'inline-grid', marginRight: '20px' }}>
-          <>
-
-            <label htmlFor="username" >Référence: </label>
-            <InputText id="username" aria-describedby="username-help" className="input-style p-inputtext-sm" />
-          </>
-          <>
-
-            <label htmlFor="username">N°compte général: </label>
-            <InputText id="username" aria-describedby="username-help" className="input-style p-inputtext-sm" />
-
-          </>
-
-          <>
-
-            <label htmlFor="username">N°compte tiers: </label>
-            <InputText id="username" aria-describedby="username-help" className="input-style p-inputtext-sm" />
-
-          </>
-
-
-        </div>
-        <div style={{ display: 'inline-grid', marginRight: '20px' }}>
-          <>
+import { Button } from "primereact/button";
 
-            <label htmlFor="username" >Libellé écriture: </label>
-            <InputText id="username" aria-describedby="username-help" className="input-style p-inputtext-sm" />
-          </>
-          <>
-
-            <label htmlFor="username">Date échéance: </label>
-            <InputText id="username" aria-describedby="username-help" className="input-style p-inputtext-sm" />
-
-          </>
-
-          <>
-
-            <label htmlFor="username">Position journal: </label>
-            <InputText id="username" aria-describedby="username-help" className="input-style p-inputtext-sm" disabled />
-
-          </>
-
-
-        </div>
-        <div style={{ display: 'inline-grid' }}>
-
-          <>
-
-            <label htmlFor="username">Débit: </label>
-            <InputText id="username" aria-describedby="username-help" className="input-style p-inputtext-sm" />
-
-          </>
-
-          <>
-
-            <label htmlFor="username">Credit: </label>
-            <InputText id="username" aria-describedby="username-help" className="input-style p-inputtext-sm" />
-
-          </>
-
-          <>
+interface VisibilityProps {
+  setProduct?: Dispatch<SetStateAction<Product>>;
+  product?: Product;
+  products?: Product[];
+  setProducts?: Dispatch<SetStateAction<Product[]>>;
+  setSelectedProduct?: Dispatch<SetStateAction<Product | undefined>>;
+  selectedProduct?: Product;
+}
 
-            <Button type='submit' label="Valider" className="input-style p-button-help" style={{ backgroundColor: '#3b82f6',border:'none',marginTop:'20px' }} />
-          </>
+interface Product {
+  jour?: string;
+  piece?: string;
+  reference?: string;
+  facture?: string;
+  type?: number;
+  compte_general?: string;
+  compte_tiers?: string;
+  libelle_ecriture?: string;
+  date_echeance?: string;
+  position_journal?: string;
+  debit?: string;
+  credit?: string;
+}
 
+export default function AddProduct(props: VisibilityProps) {
+  /* if (props.setProduct && props.selectedProduct) {
```

Comment:
why you comment that instead remove ??

Reply:
because I'm not completely finished with this feature yet